lib/imports/emails.js

```diff
@@ -72,3 +72,14 @@ export const signature = `
 `;
 
 export const signature_text = `Cheers, \nThe ${siteName} Team`;
+
+export const donation_msg = `
```

Comment:
are you able to add a link to this message?

also, do we need a separate text-only version of the message?

Reply:
just noticed your email saying this was difficulit (catching up out of order). if it's too much of a hassle, then i wouldn't worry about it. I think "it should be easy" (famous last words) to add it to the html emails, based on some of the other links I see in that file, but maybe I'm missing something. again, your call. feel free to merge as-is if you feel that's the best course of action since we want to get this change out soon

Reply:
I thought it would be easy too, but I looked at it for a bit and the easy way would be kinda messy. Basically, the code that triggers sending the message passes the URL that should be embedded in the message; the caller takes care of getting the correct base URL while the callee doesn't do anything that requires getting system parameters (to construct the proper /gear link). Not a huge deal, but I'd rather take a little time and do it right, whatever right means :-)